tests/tasks/setup_candlepin.yml

```
@@ -14,6 +14,12 @@
 
 - name: Deploy Candlepin
   block:
+    - name: Skip on OSes not supported by Candlepin
+      meta: end_play
+      when:
+        - ansible_distribution in ["CentOS", "RedHat"]
+        - ansible_distribution_major_version > '8'
```

```suggestion
        - ansible_distribution_major_version | int > 8
```
Better to "cast" the version to an `int` rather than rely on string comparison/collation
so candlepin is only supported (currently) on el7 and el8?

---

>  so candlepin is only supported (currently) on el7 and el8?

not yet, there is work ongoing though; it will be easy to test again when candlepin supports el9